guides/common/modules/proc_overriding-ansible-variables.adoc

```diff
@@ -12,42 +15,56 @@ To ensure that the variable that you override follows the correct order of prece
 .Prerequisite
 
 * You must have Ansible variables in {Project}.
-
-** To import Ansible variables, see {ConfiguringAnsibleDocURL}importing-Ansible-variables_ansible[Importing Ansible Variables]
-
-** To create Ansible variables, see {ConfiguringAnsibleDocURL}creating-Ansible-variables_ansible[Creating Ansible Variables]
-
-The following procedure makes reference to hosts and host groups.
-For more information about hosts and host groups, see the {ManagingHostsDocURL}[Managing Hosts] guide.
+For more information, see xref:importing-Ansible-roles_{context}[]
 
 .Procedure
 
 . In the {ProjectWebUI}, navigate to *Configure* > *Variables*.
 . Select the Ansible variable that you want to override and manage with {Project}.
 . Navigate to the *Default Behavior* area, and select the *Override* check box.
-. From the *Parameter Type* select the value type for validation.
-For example, a *string* or *boolean* variable.
+. In the *Parameter Type* field, select the value type for validation such as *string* or *boolean*.
+The types *array* and *hash* have further options for handling upon a variable match.
+See *Prioritize Attribute Order* below.
```

Comment:
Please reword.

Reply:
What is wrong with it?

Reply:
I was also wondering if I should include hypertext link to the step it mentions perhaps? If asciidoc let's me do that. Would that be okay?

Reply:
I guess I though it should be "For more information, see _Prioritize Attribute Order_ below.

Is `Prioritize Attribute Order` from the WebUI? If not, I would not make it bold.

Reply:
Does this work?